CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+- enable tracing of biggraphite
```

Comment:
Nitpicking: actually, this is not enabled. I'd rather say that it is _supported_.